app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/ImageRegionSelectionInteractionViewModel.kt

```diff
@@ -49,25 +54,58 @@ class ImageRegionSelectionInteractionViewModel private constructor(
       object : Observable.OnPropertyChangedCallback() {
         override fun onPropertyChanged(sender: Observable, propertyId: Int) {
           errorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
-            pendingAnswerError = null,
-            inputAnswerAvailable = answerText.isNotEmpty()
+            pendingAnswerError = pendingAnswerError, // Allow blank answer submission.
```

Comment:
This comment could move to the line where `pendingAnswerError` is initialized to explain why it is null by default.

Reply:
Comment supposed to be for `inputAnswerAvailable`. Fixed.